scout/parse/clinvar.py

```diff
@@ -0,0 +1,202 @@
+    """
+
+    clinvars = get_submission_variants(form_fields)
+    csv_lines = [] #the list containing all the lines (one line for each variant/casedata)
+    csv_line = []
+
+    for clinvar in clinvars:
+        if '##Local ID' in file_header or 'Individual ID' in file_header and 'casedata_'+clinvar in form_fields: # either the lines for a variant or the casedata lines for a variant with casedata filled in in the form
+            csv_line = [] # a line object for a single variant/casedata
+            for field in file_header:
+                if field+"_"+clinvar in form_fields: # field is provided for this variant
+                    if field == 'Reference sequence' or field == 'HGVS':
+                        ref_HGVS = form_fields['Reference sequence_'+clinvar] # capture both refseq and hgvs
+                        if field == 'Reference sequence':
+                            refseq = ref_HGVS.split('|')
+                            csv_line.append('"'+refseq[0]+'"')
+                        else:
+                            hgvs = ref_HGVS.split('|')
+                            csv_line.append('"'+hgvs[1]+'"')
+                    else:
+                        csv_line.append('"'+form_fields[field+"_"+clinvar]+'"') # capture the provided value
+                else: #field is not provided but it is provided for some other variant. Adding a blank cell for this one
+                    csv_line.append('""') # empty cell
+
+        if csv_line not in csv_lines and len(csv_line)>0:
+            csv_lines.append(csv_line)
+
+    return csv_lines
+
+
+def extract_submission_csv_lines(clinvars_dictlist):
+    """Parses a list of clinvar submission objects (variants) and creates the lines for printing
+    .Variant.csv and .CaseData.csv clinvar submission files.
+    """
+    variants_header = []
+    clinvar_lines = []
+    cdata_header = []
+    casedata_lines = []
+
+    variants_header = clinvars_dictlist[0]['variant_header']
+    casedata_header = clinvars_dictlist[0]['casedata_header']
+
+    for clinvar in clinvars_dictlist:
+        temp_var_fields = []
+        for v_field in variants_header:
+            temp_var_fields.append('"'+clinvar[v_field.replace(' ','_')]+'"')
+        clinvar_lines.append(','.join(temp_var_fields))
+
+        #collect casedata info:
+        if 'casedata' in clinvar:
+            cdata_objs = clinvar['casedata']
+            for case in cdata_objs:
+                temp_cd_fields = []
+                for cd_field in casedata_header:
+                    temp_cd_fields.append('"'+case[cd_field.replace(' ','_')]+'"')
+                casedata_lines.append(','.join(temp_cd_fields))
+
+    return '"'+'","'.join(variants_header)+'"', '"'+'","'.join(casedata_header)+'"', clinvar_lines, casedata_lines
+
+
+def create_clinvar_submission_dict(variant_header, variant_lines, casedata_header, casedata_lines, variant_types):
+    """
```

Comment:
Also nice with a lot of information here, please add `Args` and `Returns`.
I mean you could still keep this detailed information and add Args and Returns afterwards. It is nice if all docstrings has the same structure